symbols/overlay29.yml

```diff
@@ -445,6 +472,48 @@ overlay29:
         
         r0: entity pointer
         return: bool
+    - name: TryAbilityTruant
+      address:
+        NA: 0x22F97F0
+        EU: 0x22FA1FC
+      description: |-
+        Checks if an entity has the ability Truant, and if so tries to apply the pause status to it.
+        
+        r0: pointer to entity
+    - name: TeamExclusiveItemEffectIsActive
```

Comment:
Now that I know they're the same, I think I'd prefer the shorter `ExclusiveItemEffectIsActive`.